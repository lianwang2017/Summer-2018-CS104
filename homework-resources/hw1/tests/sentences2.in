big small skinny
boy girl
ran ate
fast slowly

