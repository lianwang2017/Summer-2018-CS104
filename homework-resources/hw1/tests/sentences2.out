The boy ran.
The boy ran fast.
The boy ran slowly.
The boy ate.
The boy ate fast.
The boy ate slowly.
The girl ran.
The girl ran fast.
The girl ran slowly.
The girl ate.
The girl ate fast.
The girl ate slowly.
The big boy ran.
The big boy ran fast.
The big boy ran slowly.
The big boy ate.
The big boy ate fast.
The big boy ate slowly.
The big girl ran.
The big girl ran fast.
The big girl ran slowly.
The big girl ate.
The big girl ate fast.
The big girl ate slowly.
The small boy ran.
The small boy ran fast.
The small boy ran slowly.
The small boy ate.
The small boy ate fast.
The small boy ate slowly.
The small girl ran.
The small girl ran fast.
The small girl ran slowly.
The small girl ate.
The small girl ate fast.
The small girl ate slowly.
The skinny boy ran.
The skinny boy ran fast.
The skinny boy ran slowly.
The skinny boy ate.
The skinny boy ate fast.
The skinny boy ate slowly.
The skinny girl ran.
The skinny girl ran fast.
The skinny girl ran slowly.
The skinny girl ate.
The skinny girl ate fast.
The skinny girl ate slowly.
48 sentences.
